viewer/src/main/java/nl/tailormap/viewer/stripes/ProxyRESTActionBean.java

```diff
@@ -60,6 +66,22 @@ public void checkAuthorization() {
         }
     }
 
+    @Before(stages = LifecycleStage.EventHandling)
+    public void initEndpoints() {
+        endpoints = new HashMap<>();
+        String config = context.getServletContext().getInitParameter("tailormap.restproxy.endpoints");
```

Comment:
Why use `0=/path1,1=/path2` and an additional endpoint parameter instead of just using a REST path like `@UrlBinding("/action/restproxy/{endpoint}/{path}")` and not using the `url` parameter at all (and just copying over the original parameters)? I.e. just like an Apache `ProxyPass /path1/ http://localhost/path1/` would work.

Or if you want to keep the `endpoint` and `url` parameters just use `endpoint=/path1` And just an init parameter without the equals signs, just checking if the `endpoint` parameter matches a configured value?

Reply:
More of a preference to use the same concepts (other proxyactionbean), not changing more than needed and the knowledge that this actionbean will be deleted when spring security will be used, so investing time in refactoring this seems like a waste of time